universal-login-react/src/core/constants/PaymentOptions.ts

```diff
@@ -1,6 +1,6 @@
-import {DEFAULT_GAS_PRICE, DEFAULT_GAS_LIMIT, ETHER_NATIVE_TOKEN, Message} from '@universal-login/commons';
+import {DEFAULT_GAS_PRICE, DEFAULT_GAS_LIMIT, ETHER_NATIVE_TOKEN, PaymentOptions} from '@universal-login/commons';
 
-export const transactionDetails: Partial<Message> = {
+export const paymentOptions: PaymentOptions = {
```

Comment:
Seems like it's already in `constants/gas`. Let's leave only one

Reply:
Done